index.html

```diff
@@ -4928,13 +4928,21 @@ <h3>Fitness for Purpose</h3>
     <section class="appendix informative">
       <h2>Contexts, Types, and Credential Schemas</h2>
 
-      <section class="informative">
+      <section class="normative">
         <h3>Base Context</h3>
+
+        <p class="issue" title="(AT RISK) Hash values might change during Candidate Recommendation">
```

Comment:
is there an issue we can tag to tack this?

Reply:
I don't think it's necessary to create an issue to track this statement. It creates more work for everyone w/o really having any upside. What's the issue going to say other than "the hash values might change" -- there's nothing to be discussed/debated there -- it's just a statement of fact.

Not every issue marker needs to map to an issue. This is just guidance to readers and implementers.